backend/manager/modules/bll/src/test/java/org/ovirt/engine/core/bll/RemoveUserProfilePropertyCommandTest.java

```diff
@@ -28,6 +31,9 @@ class RemoveUserProfilePropertyCommandTest extends BaseCommandTest {
     @Mock
     private DbUserDao userDaoMock;
 
+    @Mock
+    private Predicate<DbUser> isSystemSuperUserPredicate;
```

Comment:
The test condition `when(isSystemSuperUserPredicate.test(...).thenReturn(true)` seems missing for `isSystemSuperUserPredicate`

Reply:
`isSystemSuperUserPredicate.test()` returns correctly `false` as we have here no super user scenario. This scenario is tested in `AddUserProfilePropertyCommandTest.addSinglSshPropToEmptyProfileForAnotherUser`.

Reply:
@rszwajko so why declaring `isSystemSuperUserPredicate` at all if `isSystemSuperUserPredicate.test()` is not used for `RemoveUserProfilePropertyCommandTest` (and same for `UpdateUserProfilePropertyCommandTest`)?

Reply:
@sgratch 
It is used and returns `false` as we are testing a standard user scenario there.